tests/Tests/200__monitor_and_manage/220__grafana/220__test_verify_port_forwarding_grafana.robot

```diff
@@ -0,0 +1,41 @@
+*** Settings ***
+Documentation   Verify that MT-SRE can connect to Grafana-dashboard using port-forwarding
+...
+...             = Variables =
+...             | PROC             | Required |        To store process id|
+
+Library        OperatingSystem
+Library        Process
+Library        SeleniumLibrary
+Resource       ../../Resources/Page/OCPDashboard/Monitoring/Grafana.robot
+
+
+*** Variables ***
+${PROC}     none
+
+
+*** Test Cases ***
+Verify that MT-SRE can connect to Grafana using port-forwarding
+    [Documentation]  Verifies that Grafana is accessible by MT-SRE when using oc port-forwarding
+    [Tags]   Tier2
+    ...      ODS-754
+
+    # Port Forwarding
+    ${PROC} =  Enable Access To Grafana Using OpenShift Port Forwarding
+    # It takes aroung 7 sec for the website to come up
+    Sleep  7s
```

Comment:
`Enable Access To Grafana Using OpenShift Port Forwarding` could include this Sleep (with msg=Waiting until port forwarding is active)  or even better include a Wait Until ...  checking for example that this url returns data http://localhost:3001/api/health